ironfish/src/rpc/routes/wallet/buildTransaction.ts

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import * as yup from 'yup'
+import { RawTransactionSerde } from '../../../primitives'
+import { ApiNamespace } from '../namespaces'
+import { routes } from '../router'
+import { AssertHasRpcContext } from '../rpcContext'
+import { getAccount } from './utils'
+
+export type BuildTransactionRequest = {
```

Comment:
I wonder if this should be buildUnsignedTransaction?

since we have createRawTransaction and also it is returning an unsigned transaction.

Reply:
The existing endpoints are `createTransaction`, `postTransaction`, and `sendTransaction`, so I went with build to be consistent with that